Cargo.toml

```diff
@@ -6,11 +6,19 @@ authors = [
   "Jordan Santell <jsantell@mozilla.com>",
   "Joe Walker <jwalker@mozilla.com>",
   "Emily Toop <etoop@mozilla.com>",
+  "Grisha Kruglov <grigory@kruglov.ca>",
+  "Kit Cambridge <kit@yakshaving.ninja>",
+  "Edouard Oger <eoger@fastmail.com>",
+  "Thom Chiovoloni <tchiovoloni@mozilla.com>",
 ]
 name = "mentat"
-version = "0.6.0"
+version = "0.6.1"
 build = "build/version.rs"
 
+[features]
```

Comment:
I expect that you have checked that this block doesn't need to be replicated across the other `Cargo.toml` files? Currently this seems to be the only crate that enables bundled, whereas before each individual crate did.